examples/src/main/java/com/mapbox/navigation/examples/core/MultipleStopsActivity.kt

```diff
@@ -112,23 +111,6 @@ class MultipleStopsActivity : AppCompatActivity(), OnMapReadyCallback {
         )
     }
 
-    private fun initLocationEngine() {
-        val requestLocationUpdateRequest =
-            LocationEngineRequest.Builder(DEFAULT_INTERVAL_IN_MILLISECONDS)
-                .setPriority(LocationEngineRequest.PRIORITY_NO_POWER)
-                .setMaxWaitTime(DEFAULT_MAX_WAIT_TIME)
-                .build()
-
-        locationEngine?.requestLocationUpdates(
```

Comment:
remove unused code since this locationEngine isn't initialized anywhere and will always be `null`.